packages/electron-commons/test/initializers/electron-node.spec.ts

```diff
@@ -54,21 +56,37 @@ describe('onDisconnectHandler for node environment initializer', () => {
 
             dispose();
 
-            await expect(disconnectPromise).to.eventually.eq(true);
+            await expect(disconnectPromise).to.eventually.be.fulfilled;
         });
         it('should catch on env exit intentionally', async () => {
             const { disconnectPromise } = await setupRunningEnv({ errorMode: 'exit' });
-            await expect(disconnectPromise).to.eventually.eq(true);
+            await expect(disconnectPromise).to.eventually.be.fulfilled;
         });
         it('should catch on env throwing uncaught exception', async () => {
             const { disconnectPromise } = await setupRunningEnv({ errorMode: 'exception' });
 
-            await expect(disconnectPromise).to.eventually.eq(true);
+            await expect(disconnectPromise).to.eventually.be.fulfilled;
+        });
+        it('should expose disconnect reason when env throwing uncaught exception', async () => {
+            const { disconnectPromise } = await setupRunningEnv({ errorMode: 'exception' });
+
+            await expect(disconnectPromise).to.eventually.be.deep.eq({
+                exitCode: 1,
+                signal: null,
+                lastSeenError: undefined,
+            });
         });
         it('should catch on env unhandled promise rejection', async () => {
             const { disconnectPromise } = await setupRunningEnv({ errorMode: 'promiseReject' });
 
-            await expect(disconnectPromise).to.eventually.eq(true);
+            await expect(disconnectPromise).to.eventually.be.fulfilled;
+        });
+        it('should expose error when env throwing uncaught exception', async () => {
+            const { disconnectPromise } = await setupRunningEnv({ errorMode: 'exception', pipeOutput: true });
```

Comment:
I don't understand the thing with `pipeOutput`, shouldn't it work without it?

Reply:
by default stdio was ignored in these tests completely. in order to check error exposing, I done this change. anyway, refactored this